sigs/runtime-charter.md

```diff
@@ -0,0 +1,150 @@
+*   Getting network traffic into and out of workloads, or any of the interfaces related to that (e.g. CNI)
+    - these are the domain of the ([proposed](https://github.com/cncf/toc/blob/master/sigs/proposed.md)) CNCF Traffic SIG, and other related network-centric groups.  
+*   Local or remote storage associated with workloads
+     - this is the domain of the [CNCF Storage SIG](https://github.com/cncf/sig-storage) and related groups.
+
+
+## SIG Mission Statement
+
+To enable widespread and successful execution of the full spectrum of workload types,
+including both general latency-sensitive and batch, as well as more specialized
+categories listed as in scope, in cloud-native environments through:
+
+
+
+1. Providing valuable and unbiased information to the TOC,
+   End Users and Projects of the CNCF regarding areas considered in scope (see above).
+2. Collaborating effectively with other related groups (see below).
+3. Helping to maintain the continued health of the CNCF Projects deemed
+   to be within the scope of this SIG (see below)
+4. Identifying and filling gaps in the landscape of CNCF Projects within scope.
+
+Specific SIG deliverables are as per the above, and the [general SIG responsibilities
+set out by the CNCF TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs).
+
+
+## Current CNCF Projects considered to be within the Scope of this SIG
+
+
+
+1. Kubernetes
+2. Containerd
+3. Harbor
+4. Dragonfly
+5. Virtual Kubelet
+6. CRI-O
+7. KubeEdge
+8. KubeVirt
+
+
+## Interfaces With Other Related Groups
+
+
+
+*   **[Several Kubernetes SIGs](https://github.com/kubernetes/community)**
+    cover Kubernetes-specific workload, scheduling, autoscaling, execution
+    and other related abstractions, interfaces, and implementations of
+    these interfaces.  We will maintain communication with these Kubernetes
+    SIGs where needed.  Our aim is to avoid unnecessary duplication of
+    effort by the two groups and maintain clear and consistent messaging
+    to our end user community and projects.
+*   **[CNCF Security SIG](https://github.com/cncf/sig-security)**
+    - works on the more general area of cloud-native security including a
+    uthentication, authorization, encryption, accounting, auditing, and
+    related topics.  We defer as much as possible to this group to deal
+    with general security-related issues and liaise closely with them on
+    how to deal with security areas where these arise.
+*   **[CNCF App Delivery SIG](https://github.com/cncf/sig-app-delivery)**
+    is focussed on the development, deployment, operation and testing of
+    cloud-native applications.  We collaborate with this SIG where it
+    pertains to helping to ensure that the required underlying workload
+    execution abstractions and mechanisms are suitably provided to support
+    these application-level delivery needs.
+*   **Note regarding Container Registries/Repositories** and the like:
```

Comment:
Should this SIG have a relationship with OCI?

Reply:
My opinion is that the relevant projects (e.g., containerd, cri-o) should engage with OCI, or relevant Kubernetes SIGs, or other projects, or other foundations as needed.

Reply:
I think both apply.  Will update.

Reply:
Fixed.